xclim/sdba/_adjustment.py

```diff
@@ -9,21 +9,32 @@
 import numpy as np
 import xarray as xr
 
+from xclim.core.units import convert_units_to, infer_context, units
 from xclim.indices.stats import _fitfunc_1d  # noqa
 
 from . import nbutils as nbu
 from . import utils as u
+from ._processing import _adapt_freq
 from .base import Grouper, map_blocks, map_groups
 from .detrending import PolyDetrend
 from .processing import escore
 
 
+def _adapt_freq_hist(ds, adapt_freq_thresh):
+    with units.context(infer_context(ds.ref.attrs.get("standard_name"))):
+        thresh = convert_units_to(adapt_freq_thresh, ds.ref)
+    dim = ["time"] + ["window"] * ("window" in ds.hist.dims)
```

Comment:
Is this hard-coding fine? I'm assuming that hard-coding "window" at the very least is fine, because it was the case in the previous version of `_adapt_freq`. For "time", I could use `Grouper.DIM`

Reply:
Translation of `Grouper.DIM` takes place in the decorator, so it won't be understood here, directly in the function. But hardcoding `time` is fine. (and window)